content/applications/hr/lunch/products.rst

```diff
@@ -0,0 +1,59 @@
+========
+Products
+========
+
+Odoo’s *Lunch* app does not come with any products pre-configured. The individual products being
+offered must first be configured before orders can be placed.
+
+First, navigate to :menuselection:`Lunch app --> Configuration: Products`. Next, click the
+:guilabel:`New` button in the top-left corner and a blank product form loads.
+
+Enter the following information on the form:
+
+- :guilabel:`Product Name`: enter the name for the product. This field is required.
+- :guilabel:`Product Category`: using the drop-down menu, select the :ref:`category
+  <lunch/product-categories>` this product falls under. This field is required.
+- :guilabel:`Vendor`: using the drop-down menu, select the vendor that supplies this product. This
+  field is required.
+- :guilabel:`Price`: enter the price for the product. The currency is determined by the company's
+  localization.
+- :guilabel:`Description`: enter a description of the product in this field. This description
+  appears beneath the product photo when users are viewing the day's options.
+- :guilabel:`New Until`: using the calendar popover, select the date on which the product will no
+  longer be labeled as new. Until this date, a green `New` tag appears on the product.
+- :guilabel:`Company`: if the product should only be available to a specific company, select it from
+  the drop-down menu. If this field is left blank, this product is available for all companies in
+  the database.
+- Image: hover over the image box in the top-right corner of the form, and click the
```

Comment:
I totally get **NOT** applying a GUI here, but it seems a bit "disconnected" from the rest of the list...and I'm curious if you think it should get some sort of **bold** or GUI to help it fall in line... but, again, since there is no 'Image' field (specifically) present on the UI, it gets a little tricky...

Maybe this information can be added below the list OR as a specific admonition to avoid any GUI/Bolding confusion/worries? 

Not sure...totally your call. Just figured it was worth mentioning.

Reply:
This was edited to have the gui removed in a review if I am remembering correctly, but I do prefer them all being bold to match. I'll add a bold, and I'll see what @samueljlieber decides =)

Reply:
@larm-odoo and @ksc-odoo I think bold is fine to maintain a similar visual (but definitely a no on a `:guilabel:` 😉)